gapic/ads-templates/%namespace/%name/%version/%sub/services/%service/transports/rest.py.j2

```diff
@@ -269,7 +269,7 @@ class {{service.name}}RestTransport({{service.name}}Transport):
 
         {% if not method.client_streaming %}
         {% if method.input.required_fields %}
-        __REQUIRED_FIELDS_DEFAULT_VALUES: Dict[str, str] =  {
+        __REQUIRED_FIELDS_DEFAULT_VALUES: Dict[str, Any] =  {
```

Comment:
If the whole purpose of `typing` is to provide type hints, then isn't the use of `Any` defeating the purpose? Is there a more narrow type hint we could provide? (I see that the error message in [1] expects as the map value the type `"Dict[<nothing>, <nothing>]"`)

[1] https://github.com/googleapis/python-bigtable/actions/runs/4196460916/jobs/7277462654